src/Symfony/Component/Messenger/Worker.php

```diff
@@ -54,28 +57,14 @@ public function __construct(array $receivers, MessageBusInterface $bus, EventDis
      * Valid options are:
      *  * sleep (default: 1000000): Time in microseconds to sleep after no messages are found
      */
-    public function run(array $options = [], callable $onHandledCallback = null): void
+    public function run(array $options = []): void
     {
+        $this->dispatchEvent(new WorkerStartedEvent($this));
```

Comment:
Docs for available events needs a PR to add the new ones to this list: https://symfony.com/doc/current/messenger.html#messenger-events

Reply:
https://github.com/symfony/symfony-docs/pull/12586